docs/contribute/community/contributors.md

```diff
@@ -0,0 +1,25 @@
+---
+sidebar_position: 2
+---
+
+# WasmEdge Contributors
+
+We want to extend our deepest gratitude to each and every one of you who have contributed to this open-source project. Your hard work, dedication, and passion for coding have not only made this project possible but have also enriched the broader open-source community.
+
+## Your Impact
+
+- **Code Contributions**: Whether you've submitted a single line of code or have been a long-term contributor, your work has made a difference.
+- **Documentation**: Thanks to those who have improved or even just corrected a typo in the documentation. Every bit helps!
+- **Bug Reports**: Identifying issues is just as important as fixing them. Thank you for taking the time to report bugs.
+- **Feature Requests**: Your innovative ideas have been invaluable in driving the project forward.
+- **Community Support**: Those who have helped by answering questions, providing support, and moderating discussions, you've created a welcoming community for everyone.
+
+|    Name        |    Role   | Organization |  LinkedIn |  Twitter |  GitHub   |
```

Comment:
If you can't list all the maintainers, then it's meaningless.

Reply:
@alabulei1 Its a WIP, I dont have the info regarding the Role Organization and the social media links

Reply:
I don't have this information either.